### PR TITLE
Newsletter-135:Translate into Japanese

### PR DESCRIPTION
**Pull request by azuchi**

### CODE REVIEW

**Comment by bitschmidty** [comment]
@AkioNak do you plan to review the content of this translation?

**Comment by AkioNak** [comment]
@bitschmidty Yes, I will do this weekend.

File: _posts/ja/newsletters/2021-02-10-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+  今回の更新では、親のキーチェーンを使用して、GPG互換のスマートカードにロード可能なRSAベースのPGP署名鍵を作成する方法について説明しています。
+
+- [BIPs #1054][]では、Segwitのコンセンサスの変更の仕様[BIP141][]を更新し、`OP_CHECKMULTISIG`および`OP_CHECKMULTISIGVERIFY`
+  を使用した場合の署名操作（sigops）のカウント方法を明確にしました。以前の文章では、これらはP2SHを使用した時と同じようにカウントされると
+  記載されていましたが、今回の更新では直観的ではない特殊性が説明されています:
+  "公開鍵の総数が1から16の場合、CHECKMULTISIGはそれぞれ１から16のsigopsとしてカウントされ、
+  公開鍵の総数が17から20の場合、20 sigopsとカウントされます。"
+
+- [BIPs #1047][]は、フレーズから[BIP32][]シードを決定論的に生成する仕様[BIP39][]を更新し、
+  英語以外の単語リストの使用は広くサポートされていないため、実装には推奨されないという警告を追加しています。
```
Comment (AkioNak) [inline _posts/ja/newsletters/2021-02-10-newsletter.md:170]:
```suggestion
  英語以外の単語リストの使用は広くサポートされていないため実装には推奨しない、という警告を追加しています。
```

File: _posts/ja/newsletters/2021-01-27-newsletter.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ Bitcoin Coreの使用状況調査のためのリンク、Bitcoin StackExchange
 
 ## ニュース
 
-- **Taprootのアクティベーションについて話し合うために予定されたミーティング:**
+- **<!--scheduled-meeting-to-discuss-taproot-activation-->Taprootのアクティベーションについて話し合うために予定されたミーティング:**
```
[the remark this answers is not in the document]
Comment (azuchi) [reply _posts/ja/newsletters/2021-01-27-newsletter.md]: I removed this change and changed it to refer to `#taproot`.

In any case, we need to define each anchor if we want it to be the same anchor as the english version. The current anchor consists only of the english part of the data.

**Comment by azuchi** [comment]
@AkioNak Tahnks! I fixed.

**Comment by AkioNak** [comment]
@azuchi Thanks! It looks good to me.

@bitschmidty @harding @jnewbery Is this magic of generating anchors going to come true?
https://github.com/bitcoinops/bitcoinops.github.io/pull/508#discussion_r567669891

**Comment by bitschmidty** [comment]
> @bitschmidty @harding @jnewbery Is this magic of generating anchors going to come true?
> [#508 (comment)](https://github.com/bitcoinops/bitcoinops.github.io/pull/508#discussion_r567669891)

@AkioNak There is no magic yet so we will need to manual change any ja/ links for now

**Comment by bitschmidty** [comment]
I was able to build and test locally. No broken links.

**Pull request merged**